plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -10,20 +10,38 @@ import getOptionsForBrowsersByFeature from './lib/get-options-for-browsers-by-fe
 import { pluginIdHelp } from './lib/plugin-id-help';
 import { pluginHasSideEffects } from './lib/plugins-with-side-effects';
 
+const DEFAULT_STAGE = 2;
+const OUT_OF_RANGE_STAGE = 5;
```

Comment:
Love this!

Reply:
I've just added some clamping using this so if you put `144` as a stage you see the appropriate log `Stage has been disabled, features will be handled via the features option.`